pydocstringformatter/utils/output.py

```diff
@@ -0,0 +1,20 @@
+import sys
+
+
+def _encode_string(string: str) -> bytes:
+    """Encode a string to utf-8.
+
+    This can be used to circumvent the issue of the standard encoding
+    of a windows console not being utf-8.
+    See: https://github.com/DanielNoord/pydocstringformatter/issues/13
+    """
+    return string.encode("utf-8")
+
+
+def _print_to_console(string: str) -> None:
```

Comment:
```suggestion
def _print_with_emoji(string: str) -> None:
```
The probability of it being useless for something else is slim 😄

Reply:
You'll be surprised: I think the implementation of this in `click` is 100+ lines long 😅 

They handle every possible edge case, but I thought it would be good to separate every little thing into its own function if we ever need to expand on it.